axelrod/result_set.py

```diff
@@ -38,8 +39,8 @@ def __init__(self, players, interactions, repetitions=False,
         ----------
             players : list
                 a list of player objects.
-            interactions : list
-                a list of dictionaries mapping tuples of player indices to
+            interactions : dict
+                a dictionary mapping tuples of player indices to
```

Comment:
Am I correct that this is just a doc change? You haven't modified this behaviour (it was essentially incorrect).